docs/conf.py

```diff
@@ -260,6 +259,8 @@ def translate_schema_and_codelists(language='en'):
         (glob(str(codelist_source_dir / '*.csv')), codelist_target_dir, codelist_domain),
     ], localedir, language, version=os.environ.get('TRAVIS_BRANCH', 'latest'))
 
+    print("Translated schema and codelists to {}".format(language))
```

Comment:
Did you mean to leave this in? I'm not necessary against some prints if it helps people running it, just checking

Reply:
Yes, Sphinx prints some handy output during the build so I thought this wouldn't hurt.